example/next-ts/package-lock.json

```diff
@@ -1451,21 +1451,23 @@
     "@supabase/supabase-js": {
       "version": "file:../..",
       "requires": {
-        "@supabase/gotrue-js": "^1.7.1",
-        "@supabase/postgrest-js": "^0.21.1",
+        "@supabase/gotrue-js": "^1.10.2",
+        "@supabase/postgrest-js": "^0.24.1",
         "@supabase/realtime-js": "^1.0.6"
       },
       "dependencies": {
         "@babel/code-frame": {
           "version": "7.10.4",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.10.4.tgz",
+          "integrity": "sha512-vG6SvB6oYEhvgisZNFRmRCUkLz11c7rp+tbNTynGqc6mS1d5ATd/sGyV6W0KZZnXRKMTzZDRgQT3Ou9jhpAfUg==",
```

Comment:
Not sure what happned here, I just installed a package with the latest `npm@6` version...